es/ponyfill.js

```diff
@@ -0,0 +1,17 @@
+export default function symbolObservablePonyfill(root) {
+	let result;
```

Comment:
This is correct. The goal is just to provide a build that uses native language module syntax instead of CommonJS.